app/models/concerns/search_members.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module SearchMembers
+  extend ActiveSupport::Concern
+
+  included do
+    searchkick merge_mappings: true,
```

Comment:
This file looks fine to me, I'd be in favour of just disabling the CodeFactor warnings on this.